lib/class-wp-rest-sidebars-controller.php

```diff
@@ -1,463 +0,0 @@
-<?php
```

Comment:
I noticed that this file barely changed since February. So I think the version in Core in 5.7 should be enough. Regardless if it's enough or not we have a check in `load.php` that prevents the file from loading in 5.7. So I thought I'd remove but maybe I'm wrong here. cc @noisysocks 

I know we have two other endpoints in this situations: widgets and widget types but these two saw a lot of changes since 5.6 So I think we might have an error there in old versions since they're not being loaded from the plugin but the Core one is used which is probably outdated right? Should we instead rename the endpoints classnames in the plugin and override the ones used in Core (we do this for template endpoints already)

Reply:
The widgets and widgets types endpoint could use a merge from core. The ones we ship in the plugin diverged from the ones in 5.8.

As for the future changes, https://github.com/WordPress/gutenberg/issues/33810 could save us a lot of headache. For example, lately I was adding a new endpoint to `/widget-types` and went for the polyfilling path ( https://github.com/WordPress/gutenberg/pull/34230).

Reply:
This endpoint didn't ship in 5.7. We can't remove it until the minimum supported WordPress version is 5.8.